render/detailed_node.go

```diff
@@ -441,9 +459,39 @@ func getDockerLabelRows(nmd report.Node) []Row {
 }
 
 func hostOriginTable(nmd report.Node) (Table, bool) {
+	// Ensure that all metrics have the same max
+	var (
+		maxLoad  = 0.0
+		lastLoad time.Time
+	)
+	for _, key := range []string{host.Load1, host.Load5, host.Load15} {
+		if metric, ok := nmd.Metrics[key]; ok {
+			if metric.Len() == 0 {
+				continue
+			}
+			if metric.Max > maxLoad {
+				maxLoad = metric.Max
+			}
+			if lastLoad.IsZero() || metric.Last.After(lastLoad) {
+				lastLoad = metric.Last
+			}
+		}
+	}
+
 	rows := []Row{}
 	for _, tuple := range []struct{ key, human string }{
-		{host.Load, "Load"},
+		{host.Load1, "Load (1m)"},
+		{host.Load5, "Load (5m)"},
+		{host.Load15, "Load (15m)"},
+	} {
+		if val, ok := nmd.Metrics[tuple.key]; ok {
+			val.Max = maxLoad
+			val.First = lastLoad.Add(-15 * time.Second) // TODO(paulbellamy): This should be based on the duration flag, maybe? or just auto-scale to data.
```

Comment:
This defined by the -window argument to app main.  Shouldn't really hard code it.  Plumbing it though it going to be fun though....